### PR TITLE
Addition of Support Vector Model

### PR DESCRIPTION
**Pull request by mentor-mide**
This file evaluates the dataset on a support vector classification model. Please share your review. Thanks

### CODE REVIEW

**Comment by Jolomi-Tosanwumi** [comment]
That is great @Ademide-Ade. 704mins is really huge. However, we can not depend on the model's f1_score because you didn't set class_weight='balanced' during the initialization of the SVC object and this may mean that your model was biased towards the majority classes. Overall, you did a good job.

**Comment by Jolomi-Tosanwumi** [comment]
It will be great if you correct the class weight hyper-parameter. In order to reduce the time complexity of the model, don't do cross validation. Instead, do stratified train test split. Train on training set and test on the testing set. Then, find the accuracy and f1_weighted for both the training set and testing set.

Remember to set the class weights parameter of your model to 'balanced'. Also set the stratify parameter of train test split. Set random state to 1, and test_size to 0.25.

Please, effect these changes while I close this pull request.
